test/posix_time/testtime_input_facet.cpp

```diff
@@ -43,7 +43,7 @@ bool failure_test(temporal_type component,
 
 // for tests that are expected to fail quietly
 template<class temporal_type>
-bool failure_test(temporal_type component,
+bool failure_test(temporal_type& component,
```

Comment:
Why change this to a &? and if you're changing it, why not change the other overload of `failure_test` as well?

Reply:
Hi @mclow. I only needed a reference here so that I could leverage failure_test for code re-use, and call check("...", !failiure_test()) to prove something did not fail, did not set the failbit, AND store the value in component (hence the reference).